applications/StructuralMechanicsApplication/custom_utilities/explicit_integration_utilities.cpp

```diff
@@ -34,7 +34,7 @@ double CalculateDeltaTime(
     {
         "time_step_prediction_level" : 2.0,
         "max_delta_time"             : 1.0e-3,
-        "safety_factor"              : 0.4,
+        "safety_factor"              : 0.80,
```

Comment:
@KlausBSautter do you agree?

Reply:
The previous calculation was not exact because was not taking the larger side, that is not the larger edge, but the heigh of the face